packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -1480,6 +1481,67 @@ export = {
     test.done();
   },
 
+  'notifications'(test: Test) {
+    const stack = new cdk.Stack();
+    const project = new codebuild.Project(stack, 'MyCodebuildProject', {
+      buildSpec: codebuild.BuildSpec.fromObject({
+        version: '0.2',
+        phases: {
+          build: {
+            commands: [
+              'echo "Hello, CodeBuild!"',
+            ],
+          },
+        },
+      }),
+    });
+
+    project.notifyOnBuildSucceeded('NotifyOnBuildSucceeded', { target: { bind: () => ({ targetType: notifications.TargetType.AWS_CHATBOT_SLACK, targetAddress: 'SlackID' }) } });
```

Comment:
Really weird formatting here...

```suggestion
    project.notifyOnBuildSucceeded('NotifyOnBuildSucceeded', {
      target: {
        bind: () => ({
          targetType: notifications.TargetType.AWS_CHATBOT_SLACK,
          targetAddress: 'SlackID',
        }),
      },
    });
```

Same comment below.

Reply:
Already adjusted.